README.md

```diff
@@ -32,9 +32,12 @@ OR
 
 TODO: npm での使用方法を書く
 
-
 ## Usage
 
+NES.css is only providing component. You can define your layout.
```

Comment:
Proposed:

> NES.css only provides components. You're will need to define your own layout.

Reply:
* You will need to define your own layout?